Client/Frontend/BraveVPN/BraveVPN.swift

```diff
@@ -33,6 +34,8 @@ class BraveVPN {
         logAndStoreError("Failed to load vpn conection: \(error)")
       }
 
+      GRDGatewayAPI.shared()._loadCredentialsFromKeychain()
```

Comment:
Should this function be used? Usually when a function starts with `_` it is meant as `internal` and not to be used by outside code.

Reply:
does not work otherwise, i can always proxy it via helper method if needed, will add a comment for it I guess